src/reconcile/rec_write.c

```diff
@@ -2296,15 +2296,17 @@ __rec_pack_delta_leaf(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_SAVE_UPD *s
          * still include the full value in the delta? We can omit it but it will make the rest of
          * the system more complicated. Include it for now to simplify the prototype.
          */
+        /* TODO - how do we handle prepare in this case? do we pack prepare ts in start_ts? How to
```

Comment:
This will change by the data format work. I think we need a byte to indicate it is prepared for now.